x-pack/packages/security-solution/features/src/assistant/kibana_features.ts

```diff
@@ -26,7 +26,7 @@ export const getAssistantBaseKibanaFeature = (): BaseKibanaFeatureConfig => ({
   minimumLicense: 'enterprise',
   privileges: {
     all: {
-      api: [],
+      api: ['elasticAssistant'],
```

Comment:
That's the 🎟️! Good spot!

Reply:
If an existing user/role from `8.12` had this feature privilege, does this get updated automatically, or does the role need to be updated/re-applied?

Reply:
It should not impact the roles (existing or future), because this adding to the existing privilege just the tag for api availability check.

Reply:
Ahhh, perfect -- thanks for confirming!